### PR TITLE
Write Hash#fetch_values specs

### PR DESCRIPTION
**Pull request by kachick**
```shell
$ bundle exec mspec -t ruby core/hash
ruby 2.3.0dev (2015-06-12 trunk 50847) [x86_64-linux]
[\ | ==================100%================== | 00:00:00]      0F      0E

Finished in 0.064305 seconds

55 files, 367 examples, 722 expectations, 0 failures, 0 errors
```

### CODE REVIEW

File: core/hash/fetch_values_spec.rb [hunk 1]
```diff
@@ -0,0 +1,33 @@
+require File.expand_path('../../../spec_helper', __FILE__)
+require File.expand_path('../fixtures/classes', __FILE__)
+
```
Comment (eregon) [inline core/hash/fetch_values_spec.rb]: You should guard here that this is a 2.3 feature with
`ruby_version_is "2.3" do`
(and indent accordingly + `end`)

Comment (kachick) [inline core/hash/fetch_values_spec.rb]: Sure, thanks.

**Comment by eregon** [comment]
That was quick! `Hash#fetch_values` was only added 10 ago if I read right!

**Comment by eregon** [comment]
Even better, the actual implementation came today (the feature was approved 10 days ago).

**Comment by eregon** [comment]
@kachick Thanks for your contribution, it is much appreciated :+1:

**Comment by kachick** [comment]
@eregon Thanks to merge! :)

**Comment by hsbt** [comment]
@kachick Thanks for your contribution :pray:

**Comment by kachick** [comment]
@hsbt :)

**Pull request merged**